code/modules/paperwork/pen.dm

```diff
@@ -338,3 +338,13 @@
 			P.contact_poison_poisoner = user.name
 			add_attack_logs(user, P, "Poison pen'ed")
 			to_chat(user, "<span class='warning'>You apply the poison to [P].</span>")
+
+// MARK: CHAMELEON PEN
+/obj/item/pen/chameleon
+	var/forge_name = null
+
+/obj/item/pen/chameleon/attack_self(mob/living/user)
+	forge_name = tgui_input_text(user, "Enter the name of the person whose signature you want to forge", "Forge name", max_length = MAX_NAME_LEN)
```

Comment:
Please add some adjacency/stat checks on this to ensure that this can't be done while incapacitated/not adjacent

Reply:
Done